extensions/smallrye-openapi/deployment/src/test/java/io/quarkus/smallrye/openapi/test/spring/SwaggerAndOpenAPIWithCommonPrefixTest.java

```diff
@@ -25,7 +25,7 @@
     @Test
     public void shouldWorkEvenWithCommonPrefix() {
         RestAssured.when().get("/swagger-ui/index.html").then().statusCode(200).body(containsString("/swagger"));
```

Comment:
Why does swagger-ui stay on the main router? 
I understand it's an asset, but that looks not totally right.

Reply:
My reasoning, for now, is it's a Dev/Test extension only.

Happy to revise either now or in future if that's the preferred approach

Reply:
it's dev mode only because it was exposed on the _public internet_. But if we fix that "small" issue, it might be very useful even in prod.

Reply:
Fair enough, I can move to "non app" as well